Jenkinsfile

```diff
@@ -2,7 +2,7 @@
 
 properties([
     parameters([
-        string(defaultValue: 'ci40', description: 'OpenWrt branch to test against', \
+        string(defaultValue: 'ci40', description: 'OpenWrt branch to test against',
```

Comment:
This is not doing what the commit message says because half the commit is missing?